lib/plugins/transform/tokenTemplateValues.js

```diff
@@ -79,7 +79,9 @@ module.exports = superclass => class extends superclass {
     }
 
     if (this.options.json) {
-      return super.transform(JSON.parse(mustache.render(tmpFile, tokens)));
+      if (mustache.render(tmpFile, tokens)) {
+        return super.transform(JSON.parse(mustache.render(tmpFile, tokens)));
```

Comment:
Instead of rendering the template twice, would it make sense to assign the result of the render once and conditionally call the super transform?

e.g.

```javascript
const rendered = mustache.render(tmpFile, tokens);
if (rendered) {
  return super.transform(JSON.parse(rendered));
}
```